docs/source/dev/styleguide.rst

```diff
@@ -31,22 +31,53 @@ To format all files in your working copy, you can run this command in bash from
      -o -iname "*.loader" -o -iname "*.param" -o -iname "*.unitless" \
      | xargs clang-format-11 -i
 
-Format Changes Using Git
-^^^^^^^^^^^^^^^^^^^^^^^^
+Format Only Changes, Using Git
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Instead of using the bash command above you can use *Git* together with *ClangFormat* to format your patched code only.
+
+    *ClangFormat* is an external tool for code formating that can be called by *Git* on changed files only and
+    is included in the clang library.
```

Comment:
`... and is part of clang tools`

clang -format is part of clang tools, not the clang library https://clang.llvm.org/docs/ClangTools.html

Reply:
fixed